src/RemediationRoleSaver.cpp

```diff
@@ -41,3 +62,21 @@ QString RemediationSaverBase::guessFilenameStem()
     // TODO: Add guess that uses benchmark and profile names
     return QString("remediation");
 }
+
+
+BashRemediationSaver::BashRemediationSaver(QWidget* parentWindow, ScanningSession* session):RemediationSaverBase(parentWindow, session)
+{
```

Comment:
split the line above into multiple lines, break after `:` makes it more readable, indent the initialization list items and keep them one per line.